appveyor.yml

```diff
@@ -0,0 +1,43 @@
+version: 1.1.x.{build}
+skip_commits:
+  message: /(^\(?doc\)?.*|.*[A|a]cceptance [T|t]est.*)/
+clone_depth: 10
+init:
+- SET
+- 'mkdir C:\ProgramData\PuppetLabs\code && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\facter && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\hiera && exit 0'
+- 'mkdir C:\ProgramData\PuppetLabs\puppet\var && exit 0'
+environment:
+  matrix:
+  - PUPPET_GEM_VERSION: 3.7.5
```

Comment:
Add 3.4.3 to this one - note any other differences between this and https://github.com/puppetlabs/puppetlabs-powershell/pull/52/files and make sure they are included here.

I'd make a note of what versions we currently support and make sure that the oldest version is being tested.